eth-wallet/src/error.rs

```diff
@@ -0,0 +1,10 @@
+use libsecp256k1::Error;
```

Comment:
is this a copy of the conversions? can it be removed / dead code?

Reply:
This is the native `libsecp256k1` error type, we use it to generate

```rs
    TypeConversionError::InvalidLibSecp256K1TypeConversion(#[from] Error),
```

so we have a direct way to convert from one type `Error` to another